README.md

```diff
@@ -145,6 +145,9 @@ sh install_dependencies.sh
 python setup.py bdist_wheel
 pip install dist/*
 ```
+To build in CPU-only mode, set 'WITH_CUDA' to 'OFF' in [CMakeLists.txt](./CMakeLists.txt) 
+and [setup.py](./setup.py).
```

Comment:
We should not ask users to change the file. Actually changing the command to `WITH_CUDA=OFF  python setup.py bdist_wheel` works.

Reply:
fixed